### DISCUSSION TITLE
How to set an expression as a column pattern

### DISCUSSION THREAD

**Discussion by rudacs** [fluentmigrator/fluentmigrator#1688] [Q&A] [answered]
**Describe the question**
I need to define an expression **NEXT VALUE FOR [dbo].[sequenceName]**

But when you put it as WithDefaultValue it remains as text.

How do I define?

**Information (please complete the following information):**
 - OS: Windows 11
 - Platform: .NET 7
 - FluentMigrator version: 3.3.2
 - FluentMigrator runner: in-process runner
 - Database Management System: SQLServer
 - Database Management System Version: SQL Server 2022 Standard Edition

**Comment by jzabroski** [comment, marked as answer]
Direct support for retrieving the next value of a sequence would be a new feature.  

Use Execute.Sql to embed raw sql directly for now as a workaround.

Alternatively, if using an Insert or Update expression, you can use the RawSql helper.
